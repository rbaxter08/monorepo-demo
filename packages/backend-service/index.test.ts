import { dateFormatService } from ".";

describe("dateFormatService", () => {
  it("dateFormatService() should print the correct date", () => {
    const result = dateFormatService(new Date("2020-04-13T00:00:00.000+08:00"));
    expect(result).toMatchSnapshot();
  });
});
