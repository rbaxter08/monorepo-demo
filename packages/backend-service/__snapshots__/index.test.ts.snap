// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dateFormatService dateFormatService() should print the correct date 1`] = `"Sun Apr 12 2020"`;
